styles.css

```diff
@@ -12,6 +14,11 @@ body {
   list-style: none;
   padding: 0;
   margin: 0;
+  display: flex;
+}
+
+.nav-items ul > li {
+  flex: auto;
 }
```

Comment:
As an alternative, instead of adding: 

```
.nav-items ul > li {
  flex: auto;
}
```

you could have just added  `justify-content: space-between ` to the `.nav-items ul` selector.

The justify content property can be used to align the items on the main axis.